Plugwise_Smile/Smile.py

```diff
@@ -707,6 +707,8 @@ def get_appliance_data(self, dev_id):
                             continue
 
                     measure = appliance.find(pl_value).text
+                    if name == "water_pressure" and float(measure) > 3.5:
```

Comment:
Maybe add a comment that above 3.5 the value is deemed unlogic and therefor not returned?

Reply:
See the update below.